src/Plugin/WebformElement/NetsEasyPaymentElement.php

```diff
@@ -63,10 +63,12 @@ protected function defineDefaultProperties() {
    */
   public function form(array $form, FormStateInterface $form_state): array {
     $form = parent::form($form, $form_state);
+    $availableElements = $this->getRecipientElements();
     $form['element']['amount_to_pay'] = [
-      '#type' => 'textfield',
-      '#title' => $this
-        ->t('Machine name of field containing amount to pay'),
+      '#type' => 'select',
+      '#title' => $this->t('Select the element, containing the amount to pay'),
+      '#required' => FALSE,
```

Comment:
It seems weird if this can be empty:

```suggestion
      '#required' => TRUE,
```